lib/iotaUtils.js

```diff
@@ -154,6 +154,10 @@ function mergeDeviceWithConfiguration(deviceData, configuration, callback) {
             deviceData[fields[i]] = configuration[confField];
         } else if (!deviceData[fields[i]] && (!configuration || !configuration[confField])) {
             deviceData[fields[i]] = defaults[i];
+        } else {
+            config
+                .getLogger()
+                .debug(context, 'at field "' + fields[i] + '" configuration merging logic did not merge anything', i);
```

Comment:
```suggestion
                .debug(context, 'at field "' + fields[i] + '" configuration merging logic did not merge anything');
```

not fully sure but I'd say that if you don't use formating (such as "%d") then the i variable should not be included at the end of the statement.

Reply:
fixed in 9dfbdcb16e08ba11ae9b11dac785a6be24469bf6